diff --git a/tests/integration/suite/daprd/metadata/metadata.go b/tests/integration/suite/daprd/metadata/metadata.go
index 419a7ceed..d8605df75 100644
--- a/tests/integration/suite/daprd/metadata/metadata.go
+++ b/tests/integration/suite/daprd/metadata/metadata.go
@@ -100,11 +100,11 @@ func validateResponse(t *testing.T, appID string, appPort int, body io.Reader) {
 	require.NoError(t, err)
 
 	require.Equal(t, appID, bodyMap["id"])
-	require.Equal(t, "edge", bodyMap["runtimeVersion"])
+	require.Equal(t, "1.13.0", bodyMap["runtimeVersion"])
 
 	extended, ok := bodyMap["extended"].(map[string]interface{})
 	require.True(t, ok)
-	require.Equal(t, "edge", extended["daprRuntimeVersion"])
+	require.Equal(t, "1.13.0", extended["daprRuntimeVersion"])
 
 	appConnectionProperties, ok := bodyMap["appConnectionProperties"].(map[string]interface{})
 	require.True(t, ok)
