diff --git a/Makefile b/Makefile
index abcdef1..abcdef2 100644
--- a/Makefile
+++ b/Makefile
@@ -396,7 +396,7 @@ test-integration: test-deps
 		--jsonfile $(TEST_OUTPUT_FILE_PREFIX)_integration.json \
 		--format testname \
 		-- \
-		./tests/integration -timeout=30m -count=1 -v -tags="integration$(TEST_ADDITIONAL_TAGS)" -integration-parallel=false $(ARGS)
+		./tests/integration -timeout=45m -count=1 -v -tags="integration$(TEST_ADDITIONAL_TAGS)" -integration-parallel=false $(ARGS)

 .PHONY: test-integration-parallel
 test-integration-parallel: test-deps
@@ -405,7 +405,7 @@ test-integration-parallel: test-deps
 		--jsonfile $(TEST_OUTPUT_FILE_PREFIX)_integration_parallel.json \
 		--format testname \
 		-- \
-		./tests/integration -timeout=30m -count=1 -v -tags="integration$(TEST_ADDITIONAL_TAGS)" -integration-parallel=true $(ARGS)
+		./tests/integration -timeout=45m -count=1 -v -tags="integration$(TEST_ADDITIONAL_TAGS)" -integration-parallel=true $(ARGS)
