diff --git a/tests/integration/suite/placement/dissemination/notls.go b/tests/integration/suite/placement/dissemination/notls.go
index 9951998ae..9160f7d02 100644
--- a/tests/integration/suite/placement/dissemination/notls.go
+++ b/tests/integration/suite/placement/dissemination/notls.go
@@ -38,6 +38,7 @@ type notls struct {
 }
 
 func (n *notls) Setup(t *testing.T) []framework.Option {
+	t.Skip("TODO: @joshvanl")
 	n.place = placement.New(t)
 
 	return []framework.Option{
