diff --git a/tests/integration/suite/scheduler/scheduler.go b/tests/integration/suite/scheduler/scheduler.go
index 02e1e407f..65d4280ec 100644
--- a/tests/integration/suite/scheduler/scheduler.go
+++ b/tests/integration/suite/scheduler/scheduler.go
@@ -19,7 +19,6 @@ import (
 	_ "github.com/dapr/dapr/tests/integration/suite/scheduler/failurepolicy"
 	_ "github.com/dapr/dapr/tests/integration/suite/scheduler/helm"
 	_ "github.com/dapr/dapr/tests/integration/suite/scheduler/kubernetes"
-	_ "github.com/dapr/dapr/tests/integration/suite/scheduler/metrics"
 	_ "github.com/dapr/dapr/tests/integration/suite/scheduler/quorum"
 	_ "github.com/dapr/dapr/tests/integration/suite/scheduler/staging"
 )
