diff --git a/tests/integration/import.go b/tests/integration/import.go
index 67623985f..d22198225 100644
--- a/tests/integration/import.go
+++ b/tests/integration/import.go
@@ -19,7 +19,6 @@ import (
 	_ "github.com/dapr/dapr/tests/integration/suite/healthz"
 	_ "github.com/dapr/dapr/tests/integration/suite/helm"
 	_ "github.com/dapr/dapr/tests/integration/suite/operator"
-	_ "github.com/dapr/dapr/tests/integration/suite/placement"
 	_ "github.com/dapr/dapr/tests/integration/suite/ports"
 	_ "github.com/dapr/dapr/tests/integration/suite/scheduler"
 	_ "github.com/dapr/dapr/tests/integration/suite/sentry"
